src/collective/nitf/upgrades/v2002/__init__.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+from collective.nitf.logger import logger
+from collective.nitf.portlets.latest_sectionable_nitf import Assignment
+from plone import api
+from plone.portlets.interfaces import ILocalPortletAssignable
+from plone.portlets.interfaces import IPortletAssignmentMapping
+from plone.portlets.interfaces import IPortletManager
+from zope.component import getMultiAdapter
+from zope.component import getUtility
+
+
+def remove_nitf_portlet(setup_tool):
+    """Walk over the site to search and remove any portlet assignment
+    referring to the `collective.nitf.LatestSectionableNITFPortlet`.
+    """
+    def remove_assignments(obj):
+        if not ILocalPortletAssignable.providedBy(obj):
+            return
+        for name in ('plone.leftcolumn', 'plone.rightcolumn'):
```

Comment:
and Group portlets, and Content Type portlets… I'm not going to take care of that: the portlet is broken, nobody can be using it.